### ISSUE TITLE
GitHub workflow checks are not running for non-member PRs

### ISSUE THREAD

**Issue by mohsen1** [redwoodjs/redwood#358]
https://github.com/redwoodjs/redwood/pull/357 No CI

https://github.com/redwoodjs/redwood/pull/355 CI checks run

**Comment by thedavidprice** [comment]
Well, that’s no good. 🤔 

Looking into this now. Thanks!

**Comment by thedavidprice** [comment]
This was, well, frustrating to say the least. Poorly documented behavior by GitHub. Possibly a combination of settings not taking effect and needing to be reset along with Triggers not being correctly configured for public repos. Anyway, for our two Actions that run CI and Deploy Docs to Netlify:
- I turned Settings/Actions ‘enabled' to “off” and then “on”
- I deleted the Actions files and then re-added them
- I updated the CI Action specifically to trigger on pull_requests as well as push

Somewhere in there it did the trick. Tested a PR from a Fork using my personal GH Account.

Just Grrr…

But thanks, @mohsen1 This would have gone unnoticed for quite some time.
